docker-build.sh

```diff
@@ -5,6 +5,123 @@
 set -o errexit
 set -o xtrace
 
+ARCHIVE_ADDR=https://archive.ubuntu.com/ubuntu/
+PORTS_ADDR=https://ports.ubuntu.com/
+
+# Prepare HWA headers, libs and drivers for x86_64-linux-gnu
+prepare_hwa_amd64() {
+    # Download and install the nvidia headers from deb-multimedia
```

Comment:
this comment is wrong, you're getting the headers from VLC :)

Reply:
Oh. I didn't add this part, I just moved it here ;)

Reply:
I see, but it doesn't mean you cannot fix it :)